__tests__/app.test.js

```diff
@@ -124,3 +124,59 @@ describe("/api/articles", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  it("GET 200: should respond with an array of comment objects ordered by created_at (desc)", () => {
+    return request(app)
+      .get("/api/articles/9/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const result = body.comments;
+        expect(result).toBeInstanceOf(Array);
+        expect(result).toHaveLength(2);
+        expect(result).toBeSortedBy("created_at", { descending: true });
+        result.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
Ideally, because you've requested comments for article 9, you'd expect them all to have an article_id of 9, instead of any number.
`   article_id: 9,`